src/test/java/nl/atos/ontwikkelstraat/OntwikkelstraatApplicationTest.java

```diff
@@ -0,0 +1,36 @@
+package nl.atos.ontwikkelstraat;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertNotNull;
+
+/**
+ * Created by nerom on 31-3-2016.
+ */
+public class OntwikkelstraatApplicationTest {
+
+    OntwikkelstraatApplication application;
+    @Before
+    public void setUp() {
+        application = new OntwikkelstraatApplication();
+    }
+
+    @After
+    public void tearDown() {
+        application = null;
+    }
+
+    @Test
+    public void Application() {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method names should not start with capital letters](https://www.codacy.com/app/vinsterdamus/Ontwikkelstraat/file/2347129717/issues/source?bid=3138095&fileBranchId=3208141#l26)